docs/troubleshooting.md

```diff
@@ -0,0 +1,81 @@
+# Troubleshooting
+
+This page summarizes common error scenarios when running the pipeline and
+provide recommanded workarounds. If you are still unable to successfully load
+your VCF files, please post on the
+[google-genomics-discuss](https://groups.google.com/forum/#!forum/google-genomics-discuss)
+group or file a GitHub issue if you believe that there is a bug in the pipeline.
+
+## Pipeline is too slow
+
+* Try increasing `--max_num_workers`.
+* Try changing `--worker_machine_type` to a larger machine (e.g.
+  `n1-standard-32`). See
+  [predefined machine types](https://cloud.google.com/compute/pricing#predefined_machine_types)
+  for the full list.
+* Ensure you have enough [quota](https://cloud.google.com/compute/quotas) in the
+  zone/region running the pipeline. By default, the pipeline runs in the
+  `us-central1` region. You may change this by specifying `--region <region>`
+  or `--zone <zone>` when running the pipeline. You can check for quota issues
+  by navigating to the
+  [Compute Engine quotas page](https://console.cloud.google.com/iam-admin/quotas?service=compute.googleapis.com)
+  while the pipeline is running, which shows saturated quotas at the top of the
+  page.
+* `gzip` and `bzip2` file formats cannot be sharded, which considerably slows
+  down the pipeline. Consider decompressing the files prior to running the
+  pipeline. You may use [dsub](https://github.com/googlegenomics/dsub) to write
+  a script to decompress the files in scalable manner. Note that this is only an
+  issue if running the pipeline with a small number of large files (i.e. running
+  with a large number of small files is usually fine as each file can be read by
+  a separate process).
+
+See [handling large inputs](./large_inputs.md) for more details.
+
+## Pipeline crashes due to out of disk error
+
+Try increasing the disk size allocated to each worker by specifying
+`--disk_size_gb <disk_size>` and/or increasing the number of workers by
+specifying `--max_num_workers <num_workers>`. By default, each worker gets 250GB
+of disk, which may not be enough to read the entire dataset.
+
+See [handling large inputs](./large_inputs.md) for more details.
+
+## Error: "JSON parsing error ... No such field: <field_name>"
```

Comment:
This error will never happen after PR#146.

Reply:
Yeah, I think we can update the docs after the new docker image is released (it could still happen if somehow there is a bug, so I prefer to leave it, but swap the order with the one below)